sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ZonedDateTimeSerializer.java

```diff
@@ -7,31 +7,32 @@
 import com.fasterxml.jackson.databind.ser.std.StdSerializer;
 
 import java.io.IOException;
-import java.time.ZonedDateTime;
+import java.time.Instant;
+import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
 import java.util.Locale;
 
-public class ZonedDateTimeSerializer extends StdSerializer<ZonedDateTime> {
+public class ZonedDateTimeSerializer extends StdSerializer<Instant> {
```

Comment:
We should rename this class to `InstantSerializer`

Reply:
Done